.travis.yml

```diff
@@ -0,0 +1,215 @@
+language: go
+
+go:
+  - 1.13.x
+
+os: linux
+
+notifications:
+  email:
+    on_success: never
+
+cache:
+  directories:
+    - /usr/local/kubebuilder/bin
+    - $HOME/.cache/go-build
+    - $HOME/gopath/pkg/mod
+
+stages:
+  - preparation
+  - build
+  - deploy
+  - manifest
+  - release
+
+env:
+  global:
+    - GCR=false
+    - TAG=$TRAVIS_TAG
+    - VERSION_TAG="/^v.*/"
+    - DOCKER_CLI_EXPERIMENTAL=enabled
+
+######################### DEFINING BASE STEPS #########################
+x_base_steps:
+  - &build
+    script: ./build/build.sh
+    cache:
+        directories:
+          - $HOME/.cache/go-build
+          - $HOME/gopath/pkg/mod
+
+  - &deploy
+    script: ./build/deploy.sh
+
+  - &create_manifest
+    stage: manifest
+    install: skip
+    git:
```

Comment:
I guess this needs `language: minimal` or `generic` as well sinc Git clone is disabled: https://docs.travis-ci.com/user/languages/minimal-and-generic/

Reply:
Changed to generic